src/chat.rs

```diff
@@ -3637,7 +3637,7 @@ pub enum MuteDuration {
     Forever,
 
     /// Chat is muted for a limited period of time.
-    Until(SystemTime),
+    Until(std::time::SystemTime),
```

Comment:
But if we're testing, `tools::SystemTime` would be a fake empty struct anyway, so here we still need `std:time::SystemTime`. Also i'd try to minimise the difference between the release and the test configurations

Reply:
To repeat what @iequidoo said in other words:

`tools::SystemTime` isn't a struct that would be actually useful at runtime. It just exists so that you can write `SystemTime::now()`. I'm going to mark this conversation as resolved for now, since it was a misunderstanding.

This is quite confusing, but I don't have a better idea, either -  if you have a better idea @link2xt please mark as unresolved :)